src/main/java/cn/nukkit/block/BlockComposter.java

```diff
@@ -290,4 +290,4 @@ private static void registerDefaults() {
         registerBlock(65, TALL_GRASS, 2);
         registerBlock(65, TALL_GRASS, 3);
     }
-}
+}
```

Comment:
Ending newline got removed, please readd it.

Reply:
> Is there any blocking issues before we can merge this?

When placing the glow lichen, it is being positioned in the wrong direction, I still haven't been able to solve this problem, which seems to be just a visual bug